### PR TITLE
update field separators for soop_ba

### PR DESCRIPTION
**Pull request by gsatimos**
I _think_ this is right. @jonescc I updated all the elements with `FIELDSEPARATOR` to have the value as per your other harvester fix

### CODE REVIEW

File: workspace/SOOP_BA/process/Utils/CreateTalendGATT_SchemasFromMetadata2_0.1.item [hunk 1]
```diff
@@ -179,7 +179,7 @@
     </elementParameter>
     <elementParameter field="RADIO" name="USE_INLINECONTENT" value="false"/>
     <elementParameter field="TEXT" name="ROWSEPARATOR" value="&quot;\n&quot;"/>
-    <elementParameter field="TEXT" name="FIELDSEPARATOR" value="&quot;;&quot;"/>
```
Comment (jonescc) [inline workspace/SOOP_BA/process/Utils/CreateTalendGATT_SchemasFromMetadata2_0.1.item]: This one was using a semi-colon as a separator - probably better not to change ones which were valid.

File: workspace/SOOP_BA/process/Utils/CreateTalendGATT_SchemasFromMetadata2_0.1.item [hunk 2]
```diff
@@ -323,7 +323,7 @@
     <elementParameter field="TABLE" name="INTABLE"/>
     <elementParameter field="RADIO" name="USE_INLINECONTENT" value="false"/>
     <elementParameter field="TEXT" name="ROWSEPARATOR" value="&quot;\n&quot;"/>
-    <elementParameter field="TEXT" name="FIELDSEPARATOR" value="&quot;;&quot;"/>
```
Comment (jonescc) [inline workspace/SOOP_BA/process/Utils/CreateTalendGATT_SchemasFromMetadata2_0.1.item]: And this one

**Comment by gsatimos** [comment]
updated @jonescc

File: workspace/SOOP_BA/process/Utils/CreateTalendSchemasFromMetadata_0.1.item [hunk 3]
```diff
@@ -179,7 +179,7 @@
     </elementParameter>
     <elementParameter field="RADIO" name="USE_INLINECONTENT" value="false"/>
     <elementParameter field="TEXT" name="ROWSEPARATOR" value="&quot;\n&quot;"/>
-    <elementParameter field="TEXT" name="FIELDSEPARATOR" value="&quot;;&quot;"/>
+    <elementParameter field="TEXT" name="FIELDSEPARATOR" value="&quot;=>&quot;"/>
```
Comment (jonescc) [inline workspace/SOOP_BA/process/Utils/CreateTalendSchemasFromMetadata_0.1.item]: Sorry @gsatimos - a couple more like this - here and below

**Comment by gsatimos** [comment]
my bad @jonescc - thought i'd gotten them all. Should be good now

**Pull request merged**